proto/cellar.proto

```diff
@@ -0,0 +1,230 @@
+/*
+* An API for cellars that implement the EIP4626-compatible Cellar.sol contract.
+*
+* See [Cellar.sol](https://github.com/PeggyJV/cellar-contracts/blob/main/src/base/Cellar.sol)
+*/
+
+syntax = "proto3";
+package steward.v2;
+
+option go_package = "/steward_proto";
+
+/*
+* Represents a function call to a cellar that implements Cellar.sol
+*/
+message Cellar {
+    // The function you wish to execute on the target cellar
+    oneof function {
+        // Represents function `addPosition(uint256 index, address position)`
+        AddPosition add_position = 1;
+        // Represents function `pushPosition(address position)`
+        PushPosition push_position = 2;
+        // Represents function `removePosition(uint256 index)`
+        RemovePosition remove_position = 3;
+        // Represents function `setHoldingPosition(address newHoldingPosition)`
+        SetHoldingPosition set_holding_position = 4;
+        /*
+        * Represents function `rebalance(address fromPosition, address toPosition,
+            uint256 assetsFrom, SwapRouter.Exchange exchange, bytes calldata params)`
+        */
+        Rebalance rebalance = 5;
+        // Represents function `setStrategistPayoutAddress(address payout)`
+        SetStrategistPayoutAddress set_strategist_payout_address = 6;
+        // Represents function `setWithdrawType(WithdrawType newWithdrawType)`
+        SetWithdrawType set_withdraw_type = 7;
+        // Represents function `swapPositions(uint256 index1, uint256 index2)`
+        SwapPositions swap_positions = 8;
+        // Represents function `setDepositLimit()`
+        SetDepositLimit set_deposit_limit = 9;
```

Comment:
Was it decided that `SetDepositLimit` and `SetLiquidityLimit` were SP-available functions? The Notion doc seems to indicate they would have been moved to governance.

Reply:
Yes I was requested to add them. CC @josephterrigno

Reply:
Gotcha, just wanted to confirm.